README.md

```diff
@@ -532,6 +532,39 @@ ctest
 cmake --install . --prefix <path_to_install_dir>
 ```
 
+### Building for CUDA
+
+- On Linux*
+
+With the cuBLAS backend:
+
+```bash
+# Inside <path to onemkl>
+mkdir build && cd build
+export CXX=<path_to_dpcpp_compiler>/bin/dpcpp;
+cmake .. -DENABLE_CUBLAS_BACKEND=True                      \
+         -DENABLE_MKLCPU_BACKEND=False                     \   # disable Intel MKL CPU backend
+         -DENABLE_MKLGPU_BACKEND=False                     \   # disable Intel MKL GPU backend
+         [-DREF_BLAS_ROOT=<reference_blas_install_prefix>] \   # required only for testing
+         [-DREF_LAPACK_ROOT=<reference_lapack_install_prefix>] # required only for testing
```

Comment:
We don't mention anywhere above that we have lapack with cuda backend. 
And I believe we cannot run lapack with cuda backend now. Let's not mention lapack here since for cublas/curand we don't need lapack paths and we don't mention anywhere that we have support of culapack.

Reply:
Removed lapack from this command